File: src/example0/Main.java
package example0;

public class Main extends Object {

    public void run() {
        Position aPosition = new Position();
        aPosition.x = 100;

        System.out.printf("x = %d%n", aPosition.x);

        Position anotherPosition = new Position();
        anotherPosition.x = -300;

        if (anotherPosition.x < 0) {
            anotherPosition.x *= -1;
        }
        System.out.printf("x = %d%n", anotherPosition.x);
    }

    public static void main(String[] args) {
        new Main().run();
    }
}

File: src/example0/main.c
#include "position.h"

int main(void){
    
    position p;
    p.x = 100;

    printf("x = %d\n", p.x);
    
    position p2;
    p2.x = -300;
    if(p2.x < 0){
        p2.x *= -1;
    }
    printf("x = %d\n", p2.x);
    return EXIT_SUCCESS;
}
